### ISSUE TITLE
OTLP/HTTP JSON Protobuf Exporter

### ISSUE THREAD

**Issue by tsloughter** [open-telemetry/opentelemetry-erlang#122]
Right now OTLP HTTP exporter only supports exporting binary Protobufs and not in the Protobuf->JSON format. So the latter needs to be added.

**Comment by tsloughter** [comment]
No, this was never done. Only protobuf over http or grpc is supported at this time.

**Comment by sl0thentr0py** [comment]
hey is there any plan on shipping this?

**Comment by tsloughter** [comment]
There isn't. No one is working on it as far as I know.
